blog/2024-06-07-tagging-keys.mdx

```diff
@@ -0,0 +1,114 @@
+---
+slug: organize-your-translation-keys-with-tags
+title: Organize Your Translation Keys Automatically with Tags!
+description: A guide to the new Tolgee CLI feature that simplifies key organization with automatic extraction and tags.
+authors: [sgranat]
+tags: [tolgee, cli, tags, branching, tutorial]
+---
+
+Do you ever struggle with deprecated localization keys? Wondering which keys are active in production or still in draft pull requests? 😵‍💫
+
+Keeping localization keys organized in a large project can be a headache, especially with many contributors. But it’s crucial for a smooth localization process.
```

Comment:
I'm not sure if the reader understands that we are transalting Tolgee with Tolgee. How about: At Tolgee, we use our own platform to manage our localization keys, and keeping them organized in a large project can be a headache, especially with many contributors. But it’s crucial for a smooth localization process.